6/README.md

```diff
@@ -350,7 +446,7 @@ Example of an operation that failed
 }
 ```
 
-Since the `json` type for params and results allows arbitrary JSON results, implementations are free to define schemas for inputs (params) and outputs (results). Implementations should return an HTTP Bad Request response code, along with descriptive error message, in cases where the input payloads do not conform to the schema. 
+Since the `OpenAPI` type for params and results allows arbitrary results, implementations are free to define schemas for inputs (params) and outputs (results). Implementations should return an HTTP Bad Request response code, along with descriptive error message, in cases where the input payloads do not conform to the schema. 
```

Comment:
If we want to use OpenAPI, it must be optional, and must not add complexity to the core operation definition in DEP6. It is OK if this is specified as some form of additional, optional custom metadata. Please update appropriately.